guides/Getting-Started.md

```diff
@@ -231,10 +244,84 @@ main = do
 
 The `spago run` command can be used to compile and run the `Main` module:
 
-    > spago run
+    $ spago run
     [info] Build succeeded.
     The answer is 233168
 
+
+#### Compiling for the Browser
+
+Spago can be used to turn our PureScript code into JavaScript suitable for use in the web browser by using the `spago bundle-app` command:
+
+    $ spago bundle-app
+    ...
+    Build succeeded.
+    Bundle succeeded and output file to index.js
+
+All the code in the `src` directory, a standard PureScript library known as the _Prelude_ and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead-code-elimination. This `index.js` file can now be included in an HTML document. If you try this, you should see the words "Hello, World!" printed to your browser's console.
```

Comment:
We've already introduced the Prelude by this point, so I think the phrase "a standard PureScript library known as the Prelude" is unnecessary here.

Reply:
Also, while I realise it was like this in the book to start with, would you mind replacing the hyphens in "dead-code-elimination" with spaces? I don't think hyphens make sense here grammatically.